From 9ca8c2bfff884c38ce215070493d06c232c7db19 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@nic.cz>
Date: Wed, 22 Apr 2026 19:58:43 +0200
Subject: [PATCH] lpac: fix compilation error due to missing language library

Fix compilation error due to missing language library.
Fixing:
../arm-openwrt-linux-muslgnueabi/bin/ld.bfd: cannot find -lintl: No such file or directory
---
 utils/lpac/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/utils/lpac/Makefile b/utils/lpac/Makefile
index c045bac5c..a183cecae 100644
--- a/utils/lpac/Makefile
+++ b/utils/lpac/Makefile
@@ -17,6 +17,7 @@ CMAKE_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/lpac
   SECTION:=utils
@@ -26,7 +27,7 @@ define Package/lpac
     +LPAC_WITH_PCSC:libpcsclite \
     +LPAC_WITH_PCSC:pcscd \
     +LPAC_WITH_MBIM:libmbim \
-    +LPAC_WITH_UQMI:uqmi
+    +LPAC_WITH_UQMI:uqmi $(INTL_DEPENDS)
   URL:=https://github.com/estkme-group/lpac
 endef
 
-- 
2.47.3

